care/facility/migrations/0385_patientconsultation_is_readmission.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.2 on 2023-09-01 16:22
+
+from datetime import timedelta
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("facility", "0384_patientconsultation_verified_by"),
+    ]
+
+    def fill_is_readmission(apps, schema_editor):
+        PatientConsultation = apps.get_model("facility", "PatientConsultation")
+
+        bulk_update_list = []
+        qs = PatientConsultation.objects.filter(
+            suggestion="A", admission_date__isnull=False
+        )
+
+        def is_readmission(consultation):
+            # readmission=True if last_consultation.discharge_date is within 30 days of admission_date
+            return qs.filter(
+                patient=consultation.patient,
+                created_date__lt=consultation.created_date,
+                discharge_date__isnull=False,
```

Comment:
```suggestion
                created_date__lt=consultation.created_date,
                discharge_date__isnull=False,
                suggestion="A"
```
we only need to consider admissions, right?

Reply:
Yes. However, the filter was already applied in `qs` previously right.

https://github.com/coronasafe/care/pull/1572/files#diff-7f1e6658d9fb78cc639f1256f43aaefbc027080ecf1ceaded4499e3d360813a3R18